front/app/utils/permissions/rules/projectPermissions.ts

```diff
@@ -27,15 +27,8 @@ definePermissionRule(
   }
 );
 
-export function canModerateProject(
-  projectId: string | null | undefined,
```

Comment:
Made projectId required. Otherwise it doesn't make sense to check if you can moderator a project.